pygmt/base_plotting.py

```diff
@@ -318,6 +319,117 @@ def grdimage(self, grid, **kwargs):
                 arg_str = " ".join([fname, build_arg_string(kwargs)])
                 lib.call_module("grdimage", arg_str)
 
+    @fmt_docstring
+    @use_alias(
+        R="region",
+        J="projection",
+        Jz="zscale",
+        JZ="zsize",
+        B="frame",
+        C="cmap",
+        G="drapegrid",
+        N="plane",
+        Q="surftype",
+        Wc="contourpen",
+        Wm="meshpen",
+        Wf="facadepen",
+        p="perspective",
+    )
+    @kwargs_to_strings(R="sequence", p="sequence")
+    def grdview(self, reliefgrid, **kwargs):
+        """
+        Create 3-D perspective image or surface mesh from a grid.
+
+        Reads a 2-D grid file and produces a 3-D perspective plot by drawing a
+        mesh, painting a colored/gray-shaded surface made up of polygons, or by
+        scanline conversion of these polygons to a raster image. Options
+        include draping a data set on top of a surface, plotting of contours on
+        top of the surface, and apply artificial illumination based on
+        intensities provided in a separate grid file.
+
+        Full option list at :gmt-docs:`grdview.html`
+
+        Parameters
+        ----------
+        reliefgrid : str or xarray.DataArray
```

Comment:
Might be better to rename to `grid` to be consistent with the other `grd` methods.

Reply:
Done!